test/test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 coverage==4.5.2
 green==2.13.0
-pycryptodomex==3.7.2
+cryptography==2.1.4
```

Comment:
Why is this v2.1.4?

Reply:
Well spotted, I have no idea why I used that version.